### PR TITLE
Add PDA Alert modules for medical, engineering, science, and janitor teams

### PR DESCRIPTION
**Pull request by pgmzeta**
[FEATURE][STATION SYSTEMS][GAME OBJECT]
## About the PR 

* Add PDA alert modules for the Medical/Engineering/Science departments, and Janitor team.
* Add a new crisis alert for Science, complete with new sound 
* Add new sprites for both the module and the ability icon
* For each new alert module, add two of each to their respective job vendors (i.e. jobclothing / janitech).
    * Security alert modules are still only accessible via security PDAs.
* Department heads now start with an alert module for their department.

## Why's this needed? 
* Let teams know when their *own coworkers* think things are bad (and not science for the Nth time)
* Allow 'first-responder' paramedics/repairers/janitors to call for backup
* Nudge heads of staff towards directing/requesting help from their own department
 
### Item Sprites
the new modules are in the middle row:
Screenshot 2024-04-02 131634

### Ability Icons
image

### Science Alert in action
https://github.com/goonstation/goonstation/assets/91498627/d3281a23-f278-4c06-af77-50e29299ccd1

## Changelog 

```changelog
(u)glowbold
(*)Team-specific PDA Alert modules for janitorial, medical, engineering, and science are in their respective job vendors.
(+)Department head PDAs spawn with an alert module for their respective department.
```

### CODE REVIEW

**Comment by cheekybrdy** [comment]
This does sound stupid, but science missing one and ALL others kinda really exemplifies a lack of responsibility in the department. I'd like to see one at least on the shelf, or maybe for if a scientist is overwhelmed with chem alerts.

**Comment by Tarmunora** [comment]
Personally I really don't see the point but it seems harmless 🤷

File: code/modules/sound/soundCache.dm
Comment (pgmzeta) [inline code/modules/sound/soundCache.dm:1]: soundcache might not have been run in a bit? Most of those changes are unrelated.

**Comment by Glamurio** [comment]
> Mildly against as I don't really see the point and these are likely going to just be used as budget security alert panic buttons when people get jumped.

Could technically make it an interruptible action. Would still fulfill the "Need help here" function while being unusable when in combat. Honestly, I wouldn't even mind if this also applies to Security alerts, let the officer fumble on his PDA trying to hit the button as he's getting shot at.

Or could always do a testmerge.

**Comment by pgmzeta** [comment]
> Obligatory "please cite the sources of where you got the sound and if it's free to use".

I created this sound and am offering it under the default repo license of Creative Commons BY-NC-SA 3.

**Comment by TobleroneSwordfish** [comment]
Okay sorry, this is a very well made PR but we're going to have to say no. It's likely to either end up causing oversaturation of these alerts, leading to them being ignored, orr extend the RP security swarm that happens whenever you attack a secoff to at least medical doctors as well.

**Comment by pgmzeta** [comment]
> Okay sorry, this is a very well made PR but we're going to have to say no. It's likely to either end up causing oversaturation of these alerts, leading to them being ignored, orr extend the RP security swarm that happens whenever you attack a secoff to at least medical doctors as well.

would you like the new security alert sprite split out from this PR?
